docker/apply/views/help/terms-and-conditions.html

```diff
@@ -1,154 +1,216 @@
-{% extends 'layout.html' %}
+{% extends 'layout.html' %} {% block pageTitle %} Find a professional service
+abroad - Terms and conditions {% endblock %} {% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Terms and conditions</h1>
 
-{% block pageTitle %}
-  Find a professional service abroad - Terms and conditions
-{% endblock %}
+    <p class="govuk-body">These terms and conditions apply to:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Users of ‘Find a professional service abroad’</li>
+      <li>
+        Service providers who register to be included on any of the ‘Find a
+        professional service abroad’ services
+      </li>
+    </ul>
+    <p class="govuk-body">
+      By using this digital service you agree to these terms and conditions. You
+      also agreed to our
+      <a class="govuk-link" href="/help/privacy">privacy policy</a> and the
+      central
+      <a
+        class="govuk-link"
+        href="https://www.gov.uk/government/publications/foreign-and-commonwealth-office-privacy-notice-consular-services-in-the-uk-and-at-british-embassies-high-commissions-and-consulates-overseas/foreign-and-commonwealth-office-privacy-notice-consular-services-in-the-uk-and-missions-overseas"
+        >consular services privacy notice</a
+      >.
+    </p>
+    <p class="govuk-body">
+      Read our
+      <a href="/help/cookies-policy" class="govuk-link">cookies policy</a>.
```

Comment:
This is the link that needs to be changed to keep inline with the cookies link in the footer (/help/cookies) which is point 1 on the trello card

Reply:
Ah, the joys of manually editing two similar pages. I made the change [on a different T&C's page](https://github.com/UKForeignOffice/lists/pull/428/commits/776c9b540c84f1f605e186d8684e7c1a5fb45546#diff-c6f7e675659914d157b6533193a5f5234c69ac6c99cf52146980d14b98c00728R22) and must have forgotten to update this one too 🤦 

I'll get that sorted asap